src/components/ComputeModal.js

```diff
@@ -547,7 +547,17 @@ export const ComputeModalBase = ({ onDismiss, onSuccess, runtimes, persistentDis
 
   const makeImageInfo = style => div({ style: { whiteSpace: 'pre', ...style } }, [
     div({ style: Style.proportionalNumbers }, ['Updated: ', updated ? Utils.makeStandardDate(updated) : null]),
-    div(['Version: ', version || null])
+    h(Link, {
+      href: getChangelogUrl(_.find({ image: selectedLeoImage }, leoImages)),
+      disabled: !isTerraSupported(_.find({ image: selectedLeoImage }, leoImages)),
+      ...Utils.newTabLinkProps
+    },
+    ['Version: ', version || null]),
```

Comment:
Styling-wise, `[` usually comes after `},`, which would also save a little vertical space:
```suggestion
    }, ['Version: ', version || null]),
```